tests/baselines/reference/parseBigInt.errors.txt

```diff
@@ -82,13 +82,13 @@ tests/cases/compiler/parseBigInt.ts(70,72): error TS2345: Argument of type '3' i
 !!! error TS1005: ',' expected.
     { const scientific = 1e2n; }

```

Comment:
You know, on second thought, I think the `~` should cover the number _and_ the identifier. Will switch...